CODE_OF_CONDUCT.md

```diff
@@ -6,14 +6,37 @@ identity and expression, sexual orientation, disability, personal
 appearance, body size, race, ethnicity, age, religion, nationality, or
 other such characteristics.
 
-Everyone is expected to follow the [Scala Code of Conduct] when
-discussing the project on the available communication channels. If you
-are being harassed, please contact us immediately so that we can
-support you.
+The ZIO project follows the [Scala Code of Conduct](https://www.scala-lang.org/conduct/), with
+an additional clause regarding moderation that is detailed below. All participants, contributors and 
+members are expected to follow the Scala Code of Conduct when discussing the project on the available 
+communication channels. If you are being harassed, please contact us immediately so that we can support you.
```

Comment:
I would suggest if we generalize this:
```suggestion
communication channels. If the code of conduct is violated by any one, please contact one of the moderators immediately. 
```
or we can mention the discouraged behaviors to avoid that are mentioned in the CoC 
What do you think?

Reply:
I think the previous one is less formulaic and IMHO a bit more useful, because while violation of a CoC is a *proxy* for harmful behavior, harm itself is primary. We shouldn't think about violating a CoC as an offense in itself, but rather about CoC being a prescription for the community on what constitutes and how to deal with harmful behavior.

Reply:
Yes, you're right while it is mentioned that ZIO follows CoC it should be clear to respect all the statement there and the sentence like before makes sense too :).